resources/interfaces.ts

```diff
@@ -1,4 +1,9 @@
 export interface User {
+    id: number;
     email: string;
     password: string;
+}
+
+export interface UserInput {
```

Comment:
There is another way of doing this. Your ```UserInput``` has to be ```User``` without id, right ?

Reply:
yes, i probably don't even need this interface though, it's only here because "getFirstEmptyField(fields: UserInput)" looks better than how i wrote it at first "getFirstEmptyField(fields: Record<string, string>)"